File: benchmarks/c/fib.c
#include <stdio.h>

long fib(long n) {
    if (n <= 1) return n;
    return fib(n - 1) + fib(n - 2);
}

int main(void) {
    long result = fib(40);
    printf("Fib(40) = %ld\n", result);
    return 0;
}

File: benchmarks/ocaml/fib.ml
let rec fib n =
  match n with
  | 0 -> 0
  | 1 -> 1
  | _ -> fib (n - 1) + fib (n - 2)

let () =
  let result = fib 40 in
  Printf.printf "Fib(40) = %d\n" result
